maps/encounters/prisonhulk/prisonhulk.dmm

```diff
@@ -0,0 +1,5 @@
+"a" = ()
```

Comment:
This doesn't seem quite right.

Reply:
Can you clarify?

Reply:
This is an empty map -- though I'm not sure if it was intentional.

Reply:
Yeah. We're going to need to make our own. Later.